.github/workflows/integrationTest.yml

```diff
@@ -8,9 +8,9 @@ env:
   S3_INTEGRATION_BUCKET: ${{ secrets.S3_INTEGRATION_BUCKET }}
   KEY_NAME: ${{ secrets.KEY_NAME }}
   ECR_INTEGRATION_TEST_REPO: "cwagent-integration-test"
-  CWA_GITHUB_TEST_REPO_NAME: "aws/amazon-cloudwatch-agent-test"
-  CWA_GITHUB_TEST_REPO_URL: "https://github.com/aws/amazon-cloudwatch-agent-test.git"
-  CWA_GITHUB_TEST_REPO_BRANCH: "main"
+  CWA_GITHUB_TEST_REPO_NAME: "ChenaLee/amazon-cloudwatch-agent-test"
+  CWA_GITHUB_TEST_REPO_URL: "https://github.com/ChenaLee/amazon-cloudwatch-agent-test.git"
+  CWA_GITHUB_TEST_REPO_BRANCH: "securityRevision"
```

Comment:
Is this included in this PR by accident?

Reply:
No, I think she put her branch in here because PR #675 depends on [#104](https://github.com/aws/amazon-cloudwatch-agent-test/pull/104) in the testing repo. That one needs to get merged first.

Reply:
Afaics, this change is meant to setup running integration test against private fork which should be kept in the private fork. Merging this to main branch will make every integration tests in this repo run against @ChenaLee's securityRevision branch in `amazon-cloudwatch-agent-test`.